.github/workflows/release.yml

```diff
@@ -68,14 +68,12 @@ jobs:
       - name: Change dev tools to host arch (linux and macos)
         run: |
           if [[ "${{matrix.os}}" == linux* && "${{matrix.platform}}" == "aarch64" && "$(uname -m)" != "aarch64" ]]; then
```

Comment:
It might be cleaner now to use `matrix.runs-on = ubuntu*` rather than checking for `os` if the toolchain file is ubuntu specific.